### DISCUSSION TITLE
😃😃😃大佬们 可以给一些中文的注解和 使用示例嘛  想要试试呢

### DISCUSSION THREAD

**Discussion by niuxuerui666** [galacean/discussion#59] [Engine]
😃😃😃大佬们 可以给一些中文的注解和 使用示例嘛  想要试试呢     直接在vue或react引用包  就可以完成一些效果吗

**Comment by niuxuerui666** [comment]
🤣🤨现在运行不起来呢
![image](https://user-images.githubusercontent.com/62195082/107135320-b6249a00-6934-11eb-97ed-88e63548cf76.png)
![image](https://user-images.githubusercontent.com/62195082/107135326-c2105c00-6934-11eb-9d52-48573a11e35b.png)

**Comment by eyworldwide** [comment]
undefined 的变量需要 import 进来，比如：

```typescript
import  { WebGLEngine } from "oasis-engine";
```

**Comment by niuxuerui666** [comment]
有没有一些炫酷的示例呢   😋😋😊😊
还是需要自己绘制   现在的示例是这样的😋😋😋
![image](https://user-images.githubusercontent.com/62195082/107135598-fd138f00-6936-11eb-8bae-685cc1d504f0.png)

**Comment by singlecoder** [comment]
> 有没有一些炫酷的示例呢 😋😋😊😊
> 还是需要自己绘制 现在的示例是这样的😋😋😋
> ![image](https://user-images.githubusercontent.com/62195082/107135598-fd138f00-6936-11eb-8bae-685cc1d504f0.png)

https://oasis-engine.github.io/0.2/playground/#/pbr 这里有一些示例可以看看，包括实现

**Comment by niuxuerui666** [comment]
😀😘😘🥰 谢谢   我去试一下

**Comment by eyworldwide** [comment]
We should add playground links in manual.
